### ISSUE TITLE
forcePage prop is not respected 

### ISSUE THREAD

**Issue by fila90** [AdeleD/react-paginate#406]
Greetings good people maintaining this pagination lib.

From your docs on `forcePage`
```
To override selected page with parent prop. Use this if you want to control the page from your app state.
```

But this is not true. If `forcePage={5}` , for example, and you click on on page 1, page 1 becomes active instead of 5.
I see from your tests 
- `it('(observation) is not totally controlled when forcePage is provided', () => {`
- `it('(observation) is not totally controlled when forcePage is provided, even when it is 0', () => {`

that this is somewhat desirable, but not sure why?

If component is controlled, it should stay that way, there's no point having `forcePage` if it won't control the component, it behaves the same as `initialPage`

**Comment by fila90** [comment]
I've created a PR, but i can't submit it because of lack of permissions 😢

**Comment by MonsieurV** [comment]
Hi @fila90,

Please see the [v9 branch](https://github.com/AdeleD/react-paginate/tree/v9) for a true controlled version of the lib.

You can test it with:

```sh
npm install --save github:AdeleD/react-paginate#v9
```

Maybe you can link the branch you were working one so we see your suggested changes? :)

**Comment by fila90** [comment]
Is's a very simple change that gets triggered when you set new `selected` page
```js
  componentDidUpdate(prevProps) {
    if (
      this.props.forcePage !== undefined &&
      this.state.selected !== this.props.forcePage
```

I see that you're deprecating the prop so not sure if there's point in opening the PR 😅 

BTW. how should I open PR? Fork it and do it from there? I was just following your `CONTRIBUTE.md`

**Comment by MonsieurV** [comment]
Yes, for the PR, fork, then from your repo branch you can submit a PR.

I let the ticket open just to remind me to finalize the v9 release, but yes indeed, this won't be an issue anymore when we have a truly fully-controlling `page` prop.

**Comment by MonsieurV** [comment]
Maybe you can do a PR for this one?

You should look at strings with ` in the code or the [Babel configuration](https://github.com/AdeleD/react-paginate/blob/master/babel.config.js).

Do you have specific IE11 errors to share?

**Comment by fila90** [comment]
Sure, I'll take a look at it!

> Do you have specific IE11 errors to share?

Not really, it just complains that it can't find `React` for whatever reason, but it's probably some transpiling issue

**Comment by sakkelaaksonen-siili** [comment]
Any news on when v9 is gonna be released? Wanted to use this but the bug is a deal breaker, as are unreleased beta brances :sweat:

**Comment by AshuGuptaGamer** [comment]
Wait!!! so the forcepage was bugged? I am not a complete mad trying hours to debug why this was not working?

**Comment by AshuGuptaGamer** [comment]
> Hi @fila90,
> 
> Please see the [v9 branch](https://github.com/AdeleD/react-paginate/tree/v9) for a true controlled version of the lib.
> 
> You can test it with:
> 
> ```shell
> npm install --save github:AdeleD/react-paginate#v9
> ```
> 
> Maybe you can link the branch you were working one so we see your suggested changes? :)

I tried installing the branch and still have the same issue.
How do I force change pages on a button click?
```
const [page, setPage] = useState(0);

  const handlePageClick = (event) => {
    const newOffset = (event.selected * itemsPerPage) % items.length;
    console.log(
      `User requested page number ${event.selected}, which is offset ${newOffset}`
    );
    setItemOffset(newOffset);
    setPage(event.selected);
  };

<button onClick={() => setPage(2)}>Click</button>
<ReactPaginate
        marginPagesDisplayed={5}
        breakLabel='...'
        nextLabel='next >'
        onPageChange={handlePageClick}
        pageRangeDisplayed={5}
        pageCount={pageCount}
        previousLabel='< previous'
        forcePage={page}
      />
```
Some rough code

**Comment by MonsieurV** [comment]
On the v9 branch you must use the `page` prop instead of `forcePage` (look at your console when debugging, you should have see a warning indicating `forcePage` is deprecated)

**Comment by saadkhan8573** [comment]
How to run this command npm install --save github:AdeleD/react-paginate#v9 with typescript?

**Comment by pdgrimaldi** [comment]
Hi, when v9 is gonna be released? Also wanted to use this but this bug is a problem because I need to combine the paginator with a search box, so I would need to go to page 0 on each new search
